docs/markdown/Fs-module.md

```diff
@@ -194,3 +194,32 @@ Returns the last component of the path, dropping the last part of the suffix
 fs.stem('foo/bar/baz.dll')  # baz
 fs.stem('foo/bar/baz.dll.a')  # baz.dll
 ```
+
+### relative_to
+
+*since 0.57.0*
+
+Given two absolute paths, returns a version of the first path relative to the second.
```

Comment:
Why must they be absolute paths? We generally assume that a relative path is relative to the current meson.build context.

Reply:
Done